README.md

```diff
@@ -71,6 +74,15 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 
 # Useful resources on Microsoft Windows Dev Center
 | ![Academy](External/ReadMeImages/icon_academy.png) [Academy](https://developer.microsoft.com/en-us/windows/mixed-reality/academy)| ![Design](External/ReadMeImages/icon_design.png) [Design](https://developer.microsoft.com/en-us/windows/mixed-reality/design)| ![Development](External/ReadMeImages/icon_development.png) [Development](https://developer.microsoft.com/en-us/windows/mixed-reality/development)| ![Community)](External/ReadMeImages/icon_community.png) [Community](https://developer.microsoft.com/en-us/windows/mixed-reality/community)|
+
```

Comment:
I think section may have broken the rendering (at least in Chrome, it looks like this:

![image](https://user-images.githubusercontent.com/5840182/54465059-7e0b8100-4736-11e9-890d-73a21b15d6ee.png)

Reply:
And by broken rendering, I mean just like, some stuff.